src/app/access-requests/features/access-request-manager-detail/access-request-manager-detail.component.ts

```diff
@@ -91,8 +91,8 @@ export class AccessRequestManagerDetailComponent implements OnInit, HasPendingEd
 
   #cachedRequest = signal<AccessRequest | undefined>(undefined);
 
-  request = computed<AccessRequest | undefined>(
-    () => this.#cachedRequest() || this.#request.value(),
+  request = computed<AccessRequest | undefined>(() =>
+    this.#cachedRequest() || !this.#request.error() ? this.#request.value() : undefined,
```

Comment:
```suggestion
    this.#cachedRequest() || (this.#request.error() ? undefined : this.#request.value()),
```

You need the parens here because the OR binds stronger. Also I like to avoid unnecessary NOT operators.